src/components/Cards/CardDemo.scss

```diff
@@ -0,0 +1,29 @@
+@import './../../styles/fonts.scss';
+@import './../../styles/utilities/breakpoints.scss';
+
+/* stylelint-disable */
+.cards--large {
+  padding: 30px;
+  background-color: #f2edf8;
+}
+
+.cards--small {
+  padding: 30px;
+}
```

Comment:
This doesn't follow BEM styling. It should be something like this: 

```
.cards {
  &--large {
    padding: 30px;
    background-color: #f2edf8;
  }
  &--small {
    padding: 30px;
  }
}
```

Reply:
Hmm that incorrect mike, thats a stylistic choice IMO both are technically the same

Reply:
This was updated in the latest push.